.github/workflows/ci_tests.yaml

```
@@ -169,6 +169,11 @@ jobs:
         path: gatekeeper
         fetch-depth: 0 # Fetch all history for all tags and branches
 
+    - name: Change test command for exempt Namespace 
+      run: |
+        replace='num_namespaces=$(($(kubectl get ns -o json | jq '\''.items | length'\'')-$(kubectl get ns | grep kube- | wc  -l)))'
+        sed -i "185s/.*/$replace/" /home/runner/work/gatekeeper-operator/gatekeeper-operator/gatekeeper/test/bats/test.bats
```

(Optional) The hardcoded line number feels vulnerable to breaking changes.

```suggestion
        sed -i "s/num_namespaces=.*/$replace/" /home/runner/work/gatekeeper-operator/gatekeeper-operator/gatekeeper/test/bats/test.bats
```